### ISSUE TITLE
Rails 6.0 deprecation warning

### ISSUE THREAD

**Issue by kevinjbayer** [alexreisner/geocoder#1309]
I'm getting this deprecation warning, but it seems to only trigger when I'm running my Rspec tests:

DEPRECATION WARNING: Delegating missing null_value method to HighSchool. Accessibility of private/protected class methods in :scope is deprecated and will be removed in Rails 6.0.

and it's referring to this line:

 <%= f.collection_select :high_school_id, HighSchool.hs_sorted.near(current_user.zipcode), :id, :hs_name, {include_blank: true}, {:class => 'form-control'} %>

Is there just a better way I should be calling the .near method?

* Geocoder version: 1.4.9
* Rails version: 5.2
* Database (if applicable): Postgres

**Comment by craig-landry** [comment]
@kevinjbayer I think your usage is fine.  The issue is in this library and that the `near` scope defined at `lib/geocoder/stores/active_record.rb:43` calls the `null_value` method on `lib/geocoder/stores/active_record.rb:53` which is a private method defined at `lib/geocoder/stores/active_record.rb:264`.

**Comment by alexreisner** [comment]
Fixed by e2c5c1b36f3e48d7fb39bd5ff5cffcf2e8c0be82, though if anyone has a better solution I'm all ears.
